connector/connect/client/jvm/src/test/scala/org/apache/spark/sql/connect/client/SparkConnectClientSuite.scala

```diff
@@ -112,6 +130,35 @@ class SparkConnectClientSuite extends ConnectFunSuite with BeforeAndAfterEach {
     }
   }
 
+  test("SparkSession create with SPARK_REMOTE") {
```

Comment:
It's worth to also have a test covering combination of both `SPARK_REMOTE` env variable and `builder.remote(...)` call